src/main/java/net/sourceforge/tess4j/util/ImageIOHelper.java

```diff
@@ -322,71 +321,23 @@ public static ByteBuffer convertImageData(BufferedImage bi) {
         return buf;
     }
 
-    /**
-     * Gets a list of <code>IIOImage</code> objects for an image file.
-     *
-     * @param imageFile input image file. It can be any of the supported
-     * formats, including TIFF, JPEG, GIF, PNG, BMP, JPEG, and PDF if GPL
-     * Ghostscript is installed
-     * @return a list of <code>IIOImage</code> objects
-     * @throws IOException
-     */
-    public static List<IIOImage> getIIOImageList(File imageFile) throws IOException {
```

Comment:
@nguyenq why is this method being removed? I don't see the need for it, just to write it again? I don't see the need for that in here. It's just another untested unproved (in comparison to tess4j) approach, it's you call to decide.